tests/phpunit/elementor/schemas/test-usage.php

```diff
@@ -43,7 +45,36 @@ protected function validation_current_tracking_data_against_schema() {
 		return $this->run_validation_against_schema( Tracker::get_tracking_data() );
 	}
 
+	protected function generate_plugins_mock() {
+		// Arrange
+		$plugins = new Collection( [
```

Comment:
It's better to use a filter on `active_plugins` in order to test real data.

Reply:
We cannot have real data since real data based on `get_plugins()` which based on specific path and there is no filters.